manifests/vm-import-operator/v0.0.1/operator.yaml

```diff
@@ -137,7 +137,7 @@ spec:
       containers:
         - name: vm-import-operator
           # Replace this with the built image name
-          image: quay.io/masayag/vm-import-operator:latest
+          image: quay.io/masayag/vm-import-operator:v0.0.1
```

Comment:
actually - this need to be kubevirt organization - but we need permissions both for pushing image to quay.io/kubevirt and also to use 'git-release' and upload release content.